File: fibonacci.js
function fibonacci(n) {
  if (n==0) {
    return 0;
  }
  else if (n==1) {
    return 1;
  }
  else {
    return fibonacci(n-1) + fibonacci(n-2);
  }
}

console.log(fibonacci(40)) 

File: fibonacci.rs
fn fibonacci(n: u32) -> u32 {
    if n==1 {
        1
    }
    else if n==0 {
        0
    }
    else {
        fibonacci(n - 1) + fibonacci(n - 2)
    }
}

fn main() {
    println!("{}", fibonacci(40));
}
